src/Applications/LDAS_App/ldas_setup

```diff
@@ -1250,8 +1250,11 @@ class LDASsetup:
         fout =open(self.rundir+'/ldas_batchrun.j','w')
         fout.write("#!/bin/bash -f\n")
         jobid = None
+        SBATCHQSUB = 'sbatch'
         expid = self.rqdExeInp['EXP_ID']
-        fout.write("\nsed -i 's/if($capdate<$enddate) sbatch /#if($capdate<$enddate) sbatch /g' lenkf.j\n\n")
+        if self.GEOS_SITE == 'NAS':
+          SBATCHQSUB = 'qsub'
+        fout.write("\nsed -i 's/if($capdate<$enddate) "//SBATCHQSUB//" /#if($capdate<$enddate) qsub /g' lenkf.j\n\n")
```

Comment:
Based on LDAS test failures on Discover for both myself and @biljanaorescanin, I suspect this line may not work as it stands.